core/src/components/picker-column/test/columns/index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Picker Column - Standalone</title>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no"
+    />
+    <link href="../../../../../css/core.css" rel="stylesheet" />
+    <link href="../../../../../scripts/testing/styles.css" rel="stylesheet" />
+    <script src="../../../../../scripts/testing/scripts.js"></script>
+    <script nomodule src="../../../../../dist/ionic/ionic.js"></script>
+    <script type="module" src="../../../../../dist/ionic/ionic.esm.js"></script>
+    <style>
+      .grid {
+        display: grid;
+        grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+        grid-row-gap: 20px;
+        grid-column-gap: 20px;
+      }
+      h2 {
+        font-size: 12px;
+        font-weight: normal;
+
+        color: #6f7378;
+
+        margin-top: 10px;
+        margin-left: 5px;
+      }
+    </style>
+  </head>
+  <script type="module">
+    import { pickerController } from '../../../../dist/ionic/index.esm.js';
+    window.pickerController = pickerController;
+  </script>
+
+  <body>
+    <div class="grid ion-padding">
+      <div class="grid-item">
+        <h2>Single Column</h2>
+        <ion-button onclick="removeOption()" id="single-column-remove-button">Remove Option</ion-button>
+        <ion-button onclick="addOption()" id="single-column-add-button">Add Option</ion-button>
+      </div>
+      <div class="grid-item">
+        <h2>Multiple Columns</h2>
+        <ion-button onclick="removeOption(2, 5, multiColumnOptions)" id="multiple-column-remove-button"
+          >Remove Option</ion-button
+        >
+        <ion-button onclick="addOption(2, 5, multiColumnOptions)" id="multiple-column-add-button"
+          >Add Option</ion-button
+        >
+      </div>
+    </div>
+    <script>
+      const defaultColumnOptions = [['Dog', 'Cat', 'Bird', 'Lizard', 'Chinchilla']];
+
+      const multiColumnOptions = [
+        ['Minified', 'Responsive', 'Full Stack', 'Mobile First', 'Serverless'],
+        ['Tomato', 'Avocado', 'Onion', 'Potato', 'Artichoke'],
+      ];
+
+      async function removeOption(numColumns = 1, numOptions = 5, columnOptions = defaultColumnOptions) {
+        const picker = await pickerController.create({
+          columns: this.getColumns(numColumns, numOptions, columnOptions),
+          buttons: this.getButtons(),
+          animated: false,
```

Comment:
Why the `animated: false`? I would rather leave as many options at the default as possible, especially since we're changing the `noAnimate` behavior so it would be good to be able to check that the animations still work at a glance.